docs/architecture.md

```diff
@@ -112,76 +112,206 @@ The redux store is composed of two `slices` : `documents` and `queries`:
 
 }
 ```
-> ℹ️ If queries are not named, a name is automatically generated but this means that the `queries`
-slice can grow indefinitely if there are a large number of queries. This is why you are 
-encouraged to name your queries : `client.query(qdef, { as: 'finishedTodos'})`. 
+> ⚠️ If queries are not named, a name is automatically generated, but this means that the `queries`
+collection can grow indefinitely. This is why you are encouraged to name your queries : `client.query(qdef, { as: 'finishedTodos'})`. 
+See these [naming rules](https://github.com/cozy/cozy-guidelines?tab=readme-ov-file#naming-of-queries) that are followed at Cozy.
 
-The glue between the Redux store and the UI is done via ObservableQuery.
-`ObservableQuery` are objects instantiated by a <Query /> component. Their
-role is to react to store changes and wake the component. They should not
-be used directly as `useQuery` and `queryConnect` do the job for you.
+ℹ️ See the [react integration](/docs/react-integration.md) for more insight about the glue between the redux store and the UI.
 
 What do we mean exactly by saying "This redux store
 brings observability to cozy-client, and allows for connection of
 UI components to the data." Let's take a full exemple: 
 
-We have a component that displays a todolist: 
+Here is a simple React example on how to display data with cozy-client:
 ```jsx
-// TodoListComponent.jsx
 const { data, fetchStatus } = useQuery(Q('io.cozy.todos'), {'as': 'todoslist'})
+if(fetchStatus === 'loading'){
+  return <Spinner /> 
+}
 if(fetchStatus === 'loaded'){
   return <TodoLists todos={data}> /> 
 }
 ```
 
-But we also have a component that gives use the opportunity to add a 
-new todo: 
-```jsx
-client.save({_type: 'io.cozy.todos', 'label': 'New TODO'});
+This way, a spinner will be displayed during the time the query is actually run. Once the query is done, data is available and can be displayed.
+
+Any data change will be automatically handled by cozy-client.
```

Comment:
It can be hard to understand that sentence. We should talk about the data concerned by the queries, not all the data. 

If you're still on the exemple, then having `Any data change on the io.cozy.todos doctype, will be automatically handled by cozy-client`. 

But I'm wondering also, what does `automatically handled` mean.

Reply:
You should link to `Auto query update`

Reply:
> But I'm wondering also, what does automatically handled mean.

right, it's a bit an easy word, I removed it